test/test.cc

```diff
@@ -2881,6 +2881,17 @@ TEST_F(ServerTest, GetStreamedChunkedWithGzip) {
   EXPECT_EQ(std::string("123456789"), res->body);
 }
 
+TEST_F(ServerTest, GetMethodWithParams) {
+  Params params;
+  params.emplace("test1", "param1");
+
+  auto res = cli_.Get("/getmethodwithparams", params);
```

Comment:
I don't see any corresponding `/getmethodwithparams` in `ServerTest::SetUp` where you have to test the given parameters.

You could refer to the following:
https://github.com/yhirose/cpp-httplib/blob/301faa074c4a0fa1dbe470dfb4f77912caa1c57f/test/test.cc#L1448-L1456